LexikJWTBundle | Make commands

Hi everyone, 

Just trying to install the Lexik/jwt-authentication-bundle but after recipes configuration, the console displays the make commands : 

```bash
# composer req jwt-auth
Using version ^2.4 for lexik/jwt-authentication-bundle
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 2 installs, 0 updates, 0 removals
  - Installing namshi/jose (7.2.3): Downloading (100%)         
  - Installing lexik/jwt-authentication-bundle (v2.4.2): Downloading (100%)         
Writing lock file
Generating autoload files
Symfony operations: 1 recipe (fcdf4c052de7b1c2a71ca27807b24266)
  - Configuring lexik/jwt-authentication-bundle (2.3): From github.com/symfony/recipes:master
ocramius/package-versions:  Generating version class...
ocramius/package-versions: ...done generating version class
Executing script cache:clear [OK]
Executing script assets:install --symlink --relative public [OK]

 Next for LexikJWTAuthenticationBundle 
    Run make generate-jwt-keys to generate a key pair.

# make generate-jwt-keys
make: *** No rule to make target 'generate-jwt-keys'.  Stop.
```

I've try to use the CLI but it doesn't display the "classical commands of the bundle", I can use the "bash" approach but that's not really "logic" for newcomers : 

```bash
mkdir -p config/jwt
openssl genrsa -passout pass:${JWT_PASSPHRASE} -out ${JWT_PRIVATE_KEY_PATH} -aes256 4096
openssl rsa -passin pass:${JWT_PASSPHRASE} -pubout -in ${JWT_PRIVATE_KEY_PATH} -out ${JWT_PUBLIC_KEY_PATH}
```

It there any plan on installing the bundle commands in the classical CLI?

--- a/lexik/jwt-authentication-bundle/2.3/post-install.txt
+++ b/lexik/jwt-authentication-bundle/2.3/post-install.txt
@@ -1,2 +1,0 @@
-<bg=blue;fg=white> Next for LexikJWTAuthenticationBundle </>
-    Run <comment>make generate-jwt-keys</> to generate a key pair.

